api/models/news.py

```diff
@@ -0,0 +1,25 @@
+"""Import Django models."""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from .base import Base
+
+
+class News(Base):
```

Comment:
Why not inherit from `api_textual.AbstractText` and place this to `api_textual` application?